algebird-core/src/main/scala/com/twitter/algebird/QTree.scala

```diff
@@ -367,3 +367,9 @@ case class QTree[A](
     (n.toDouble(ll) / luc, n.toDouble(uu) / ulc)
   }
 }
+
+case class QTreeAggregator(percentile: Double, k: Int) extends Aggregator[Double, QTree[Double], (Double, Double)] {
+  def prepare(input: Double) = QTree(input)
+  def semigroup = new QTreeSemigroup[Double](k)
+  def present(qt: QTree[Double]) = qt.quantileBounds(percentile)
```

Comment:
Should there be a version (maybe even the default version) that just picks the lower or upper bound here?